spec/models/site_spec.rb

```diff
@@ -344,5 +344,22 @@ def expect_correct_themes(guardian)
         &modifier_block
       )
     end
+
+    describe "experimental_content_localization" do
+      it "returns localized category names when enabled" do
```

Comment:
I think it is also worth testing the case where `experimental_content_localization` is enabled but there are no localizations and that the untranslated name and description is returned.